extensions/azure-extensions/src/main/java/io/druid/storage/azure/AzureLoadSpec.java

```diff
@@ -31,24 +31,24 @@
 public class AzureLoadSpec implements LoadSpec
 {
   @JsonProperty
-  private final String storageDir;
+  private final String blobPath;
 
   private final AzureDataSegmentPuller puller;
 
   @JsonCreator
   public AzureLoadSpec(
-      @JacksonInject AzureDataSegmentPuller puller,
-      @JsonProperty("storageDir") String storageDir
```

Comment:
Looking a bit more at the azure `CloudStorageAccount` api, it seems the concept of account is very much tied to the connection parameters and authentication, so I think it makes sense to only have `container` and `blob` parameters as part of the loadSpec.